### ISSUE TITLE
Strange behaviour renaming request in sidebar when clicking on title

### ISSUE THREAD

**Issue by liam-nt** [getinsomnia/insomnia#6803]
### Expected Behavior

Typing a new name for request updates the existing name

### Actual Behavior

Pressing the C key fills the textbox with the request URL

### Reproduction Steps

1. Add new request from curl string
2. Click title in sidebar to rename
3. Delete existing title
4. Pressing the 'c' key immediately fills the title with the request URL
5. Continuing to type, the request URL occasionally replaces the new name you've been typing

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

8.4.0

### What operating system are you using?

macOS

### Operating System Version

macOS Ventura 13.4.1

### Installation method

Download from insomnia

**Comment by subnetmarco** [comment]
We are changing the single click to double click in 8.4.2 tomorrow. We did that to be consistent with how tests are being renamed, but it is obvious it should have been the other way around.

**Comment by subnetmarco** [comment]
[v8.4.2](https://github.com/Kong/insomnia/releases/tag/core%408.4.2) has been released and this issue has been resolved.

**Comment by liam-nt** [comment]
Issue still happening in v8.4.2, 

Version: Insomnia 8.4.2
Build date: 11/15/2023
OS: Darwin arm64 22.5.0
Electron: 27.0.3
Node: 18.17.1
Node ABI: 118
V8: 11.8.172.16-electron.0
Architecture: arm64

**Comment by humanscape-reo** [comment]
@subnetmarco Hi, I don't think it is about the double-click change.
When renaming request in sidebar, specific key input(`D` in my case) makes it lose focus and cancel renaming, just like if I press escape button.
It's so annoying and wondering if there's any plan to fix this.


Version: Insomnia 8.4.5
Build date: 11/24/2023
OS: Darwin arm64 23.1.0
Electron: 27.0.3
Node: 18.17.1
Node ABI: 118
V8: 11.8.172.16-electron.0
Architecture: arm64

**Comment by subnetmarco** [comment]
@humanscape-reo this is the behavior I see in 8.5.0-beta: https://www.loom.com/share/6b3f1957fa0540998cdad82db21627af

I seem to be able to press "D". Do you confirm that this is **not** what you are experiencing?

**Comment by humanscape-reo** [comment]
@subnetmarco sorry for the unclear detail, I captured a video if it can help. I was trying to erase name and input `d`, but the input ends.

https://github.com/Kong/insomnia/assets/136434145/26dcaa55-e1b4-4c5b-a30b-c69eb7505137

and more details:
- The key that causes issue is always different. I suffered it with `m` last week, `d` in this morning, ...
- When I input other character which is ok first, I can input the rest of keys. e.g. when I lose my focus when press `d`, I can input `sddasd`.
- start renaming with enter is _ususally_ ok, but the issue occurs again when repeat several times.
